### DISCUSSION TITLE
Bitmap issue

### DISCUSSION THREAD

**Discussion by thedanniel** [ZinggJM/GxEPD2#75] [Q&A] [answered]
I was able to succesfull show bitmaps on my 7 color 7.3in acep display via wifi and SD card examples. But for some reason, the image of Brazil flag wasn't displayed properly
![IMG_20241221_200807](https://github.com/user-attachments/assets/8045e9c5-1f4a-4516-a985-104fa399c394)

I opened the image in Photoshop, cropped to 800x480 dimensions, used export to legacy, save in GIF format with 7 color table, converted the GIF to 16 bit BMP and uploaded to github repo where it would be downloaded by wifi example. I did the same procedure for the giraffe and other pictures and all were successful, but for some reason, the flag shows like this
![IMG_20241221_202052](https://github.com/user-attachments/assets/74e22808-3bc6-4eb8-91fd-f7e3d1dc1ca5)

I don't know if I did something wrong or if it is an issue with code/library, so I preferred to open a discussion instead of an issue

**Comment by G6EJD** [comment]
That's not a GxEPD issue that's an image format issue, so you need to correct the source image.

**Comment by thedanniel** [comment]
I used a new picture and the picture again looks nice, so I redid the same steps with the brazilian flag but the flag looks the same as before, I don't understand, if I was doing something wrong, all pictures would look wrong. I tried both 16 colors and 24 bit BMPs
![IMG_20241222_113103](https://github.com/user-attachments/assets/9ac2305b-e481-4b57-a0d7-378a20e2bece)

**Reply by G6EJD** [reply]
Have you turned interlace off?

**Reply by thedanniel** [reply]
@G6EJD yes, the interlacing checkbox is unchecked by default

**Comment by ZinggJM** [comment, marked as answer]
I need to know which 7.3" panel you have, there should be a sticker on the backside, and which driver class you select to compile for.
And if you use the actual version of GxEPD2; there was a fix added for GDEP073E01 in version 1.6.0.

**Reply by thedanniel** [reply]
Here it is
![IMG_20241223_051503](https://github.com/user-attachments/assets/7919f8ce-6308-419d-82ba-ea1eafa4941b)

**Reply by thedanniel** [reply]
@ZinggJM I'm using GxEPD2_7C < GxEPD2_730c_GDEY073D46, GxEPD2_730c_GDEY073D46::HEIGHT / 4 > display(GxEPD2_730c_GDEY073D46(/*CS*/ 15, /*DC=*/ 16, /*RST=*/ 17, /*BUSY=*/ 14));

**Comment by thedanniel** [comment]
I updated to 1.6.1 and it worked, the flag looks correct now🙏🏻

![IMG_20241223_060227](https://github.com/user-attachments/assets/1be1bf1c-aed0-4bfb-afb4-d342e573cf14)

**Comment by ZinggJM** [comment]
Good to know that you got it working.

But I don't understand how the update did help with your display SKU23433 (F).

Maybe you could report the driver class you selected once more.
For me to understand, maybe, and for any other user interested also.

**Reply by thedanniel** [reply]
here it is once more
![image](https://github.com/user-attachments/assets/34f62cd5-f585-4c9b-b0bf-f2d7bbeba70c)

**Reply by thedanniel** [reply]
@ZinggJM now that you mentioned, I remember I tried driver GxEPD2_730c_ACeP_730 when the display arrived and I was learning how to use it, but it gives me wrong colors
![IMG_20241223_101923](https://github.com/user-attachments/assets/296badcc-f32a-43bf-8328-2920c1a3a587)
colors
